src/components/CollapsibleTable.js

```diff
@@ -144,11 +163,20 @@ const CollapsibleRow = (props) => {
             {open ? <KeyboardArrowUpIcon /> : <KeyboardArrowDownIcon />}
           </IconButton>
         </StyledTableCell>
-        {row.data.map((cell, i) => (
-          <StyledTableCell key={i}>
-            {cell.value}
-          </StyledTableCell>
-        ))}
+        {row.data.map((cell, i) => {
+          if (!('truncate' in cell)) {
+            cell.truncate = false;
+          }
```

Comment:
Is this check needed? I think the `cell?.truncate` should take care of the condition.

Reply:
Same feedback that @fboulnois has here, I don't think this is necessary.